AdyenTests/Card Tests/Utilities/CardEncryptorCardTests.swift

```diff
@@ -196,20 +196,14 @@ class CardEncryptorCardTests: XCTestCase {
 
         var payload = Data()
         let expected: Data
-        if #available(iOS 13.0, *) {
```

Comment:
Nope, this will fail on iOS 12 pipeline that we run on master

Reply:
the whole function has `@available(iOS 13.0, *)` iOS 12 won't execute this test case, right?.

Xcode was complaining with a warning.

Reply:
nope, It executes no matter what :(

Reply:
reverted the change.